utils/rendering/controls-template.html

```diff
@@ -30,7 +30,9 @@ <h2>{{{ control.id }}}: {{{ control.title }}}</h2>
     {{%- endif -%}}
     {{%- else %}}
     {{%- if selection in rules %}}
-    <li><a href="https://github.com/ComplianceAsCode/content/tree/master/{{{ rules[selection].relative_definition_location }}}">{{{ selection }}}</a>: {{{ rules[selection].title }}}</li>
+    <li><a href="https://github.com/ComplianceAsCode/content/tree/master/{{{ rules[selection].relative_definition_location }}}"> {{{ selection }}}</a>: {{{ rules[selection].title }}}
+      <ul><li>{{{ rules[selection].identifiers['cce'] }}}</li></ul>
```

Comment:
You should add an if guard `rules[selection].identifiers['cce']` so that if rule or product don't use CCE it doesn't leave a blank bullet point.